repo/repo.go

```diff
@@ -0,0 +1,40 @@
+package repo
+
+import (
+	"fmt"
+	"github.com/dgraph-io/badger/v3"
+	"github.com/expect-digital/translate/pkg/model"
+	"log"
+)
+
+type Repo struct {
+	db *badger.DB
+}
+
+func NewRepo(db *badger.DB) *Repo {
+	return &Repo{
+		db: db,
+	}
+}
+
+func Connect() (*badger.DB, error) {
+	db, err := badger.Open(badger.DefaultOptions("").WithInMemory(true))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return db, nil
+}
+
+func (repo *Repo) SaveMessages(m model.Messages) error {
+	for _, value := range m.Messages {
```

Comment:
`Messages` is a resource to be saved. Serialize it to JSON. The key is translation id.